workflows/templates/kebechet-job.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: kebechet-job
+  annotations:
+    thoth-station.ninja/template-version: 0.1.1
+  labels:
+    app: thoth
+    component: kebechet-job
+
+spec:
+  templates:
+    - name: kebechet-job
+      inputs:
+        parameters:
+        - name: WEBHOOK_PAYLOAD
+        - name: IMAGE_STREAM_REGISTRY
+        - name: IMAGE_STREAM_NAMESPACE
+        - name: IMAGE_STREAM_TAG
+      container:
+        image: "{{inputs.parameters.IMAGE_STREAM_REGISTRY}}/\
+        {{inputs.parameters.IMAGE_STREAM_NAMESPACE}}/\
+        kebechet-job:{{inputs.parameters.IMAGE_STREAM_TAG}}"
+        command: [sh, -c]
+        args: [
+            '
+            ./app.sh
+            ',
+          ]
```

Comment:
Is this needed? I think the Dockerfile already states this (and we should slowly migrate to s2i if possible).

Reply:
Umm it seems to not trigger the job, unless this is stated, I can try again.

Reply:
I have removed it now.

Reply:
Does it work even if removed?

Reply:
@fridex It triggers the job.